Docs/README.md

```diff
@@ -64,65 +72,101 @@ Splice reads:                                  4161455
+##subread featureCounts 
+The featureCounts program from the subread package summarises the read distribution over genomic features such as genes, exons, promotors, gene bodies, genomic bins and chromosomal locations. 
+
+In addition we also plots the Biotype information, which includes miRNA, rRNA, diffferent kinds of pseudogenes to mention a few. 
+In both plot you generally want the blue bar to be as big ass possible. That means assigned feature in the general featureCounts plot and protein coding in the Biotype plot. The biotype plot is a useful tool for detecting rRNA contamination in your samples. 
+
+<img src="images/featureCounts_assignment_plot.png">
+<img src="images/featureCounts_biotype_plot.png">
+ 
 ##String Tie
-FPKMs for genes and transcripts
+StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts. It uses a novel network flow algorithm as well as an optional de novo assembly step to assemble and quantitate full-length transcripts representing multiple splice variants for each gene locus.
+The output from stringtie is FPKMs for genes and transcripts. The analysis generates the following files: 
+_\<sample\>_ligned.sortedByCoord.out.bam_transcripts.gtf
+_\<sample\>_Aligned.sortedByCoord.out.bam_transcripts.gtf
+_\<sample\>_Aligned.sortedByCoord.out.bam.cov_refs.gtf
+
 ## edgeR
-edgR is a bioconductor package for R and is used to do differntial extression analysis on RNAseq data.
+edgR is a bioconductor package for R and is used to do differential extression analysis on RNAseq data.
 Rcreate MDS plot and sample pairwise distance heatmap / dendrogram
 [edgeR Bioconductor page](https://bioconductor.org/packages/release/bioc/html/edgeR.html)
 The output generated from edgeR process is a heatmap and a MDS-plot.
-The heatmap may look something like this:
-<img src= "images/heatmap.png" width="500" height="500" >
+The heatmap may look something like this;
+
+**Heatmap:**
+<img src= "images/mqc_hcplot_hocmzpdjsq.png" >
+**MDS plot:**
+<img src= "images/mqc_hcplot_ltqchiyxfz.png" >
+The hatmap and MDS plot together try and give you a feeling for which of your sample libraries that are most simmilar to eah other. 
 
 ##MultiQC
-[MultiQC](http://multiqc.info/) is a visualation tool and is what we at NGI use to generate our hmtl reports. It collects all the reports for your samples and genearate clear and simple reports for all your samples for an easy overview of your project. All the results of the RNA seq BP pipeline 2.0 is available in the `multiqc_report.html` file.  
+[MultiQC](http://multiqc.info/) is a visualation tool and is what we at NGI use to generate our hmtl reports. It collects all the reports for your samples and genearate clear and simple reports for all your samples for an easy overview of your project. Almost all of the output results of the RNA seq BP pipeline 2.0 is available in the `multiqc_report.html` file.  
```

Comment:
A few typos, reads a bit strangely.. Also the filename will probably be different because it'll have the project name.

> [MultiQC](http://multiqc.info) is a visualisation tool that generates a single HTML report summarising all samples in your project. Most of the pipeline QC results are visualised in the report and further statistics are available in within the report data directory.